### DISCUSSION TITLE
How do we tell hosted renovate that we're using self-hosted now for one repository

### DISCUSSION THREAD

**Discussion by rbtcollins** [renovatebot/renovate#26082] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

We have renovate hosted app configured for all our repositories in github - thats desirable so we don't need to remember to onboard it for new repos.

However we have one repo that has to run code-generation (bazel glue) on every update and that is now self-hosted.

We have some additional credentials in the renovate.json5 for this self-hosted repository, and now the hosted app throws an error about the encryption key being wrong on every run.

What we'd like to do is to be able to tell the hosted app to not even try to run; as the self hosted one is working great. For this one repo. The rest being hosted is much better as they run on demand, very nice etc.

A label on the bug that renovate opens to stop it reopening it would be one way, for instance.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rbtcollins** [comment]
Logs:
```
INFO: Repository has invalid config
{
  "error": {
    "validationError": "Failed to decrypt field password. Please re-encrypt and try again.",
    "message": "config-validation",
    "stack": "Error: config-validation\n    at decryptConfig (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/config/decrypt.ts:192:27)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at decryptConfig (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/config/decrypt.ts:238:13)\n    at mergeRenovateConfig (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/repository/init/merge.ts:259:27)\n    at getRepoConfig (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/repository/init/config.ts:12:12)\n    at initRepo (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/repository/init/index.ts:52:12)\n    at Object.renovateRepository (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/repository/index.ts:55:14)\n    at attributes.repository (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/global/index.ts:184:11)\n    at start (/opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/workers/global/index.ts:169:7)\n    at /opt/containerbase/tools/renovate/37.59.8/node_modules/renovate/lib/renovate.ts:18:22"
  }
}

DEBUG: raiseConfigWarningIssue()
DEBUG: getBranchPr(renovate/configure)
DEBUG: findPr(renovate/configure, undefined, open)
DEBUG: findPr(renovate/configure, undefined, closed)
DEBUG: ensureIssue(Action Required: Fix Renovate Configuration)
DEBUG: Retrieving issueList
DEBUG: Retrieved 2 issues
DEBUG: Issue is open and up to date - nothing to do
DEBUG: Repository result: config-validation, status: onboarded, enabled: true, onboarded: true
```

**Comment by rarkins** [comment]
Try this:
- Go to the repo in developer.mend.io
- Click the Settings icon (need to be an admin)
<img width="121" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/d677a62e-5113-4cf1-ba6b-eccb783afbb9">

- Change the Renovate mode from Interactive to Silent: 
<img width="762" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/c3ef7f5b-8c7e-4d37-b5b2-312cb98ee488">

- Click Save

Renovate might still run against the repo, but it will only do a dry run and not post any issues or PRs.
